scripts/setup.sh

```diff
@@ -185,6 +185,57 @@ if [ ! -f /var/goddard/setup.lock ]; then
 
 		fi
 
+		# done
+		echo "{\"build\":\"busy\",\"process\":\"\",\"timestamp\":\"$( date +%s )\"}"  > /var/goddard/build.json
+
+		# update in file
+		running_docker_status=$(docker ps)
+
+		# clean up the docker output
+		cleaned_docker_status=$(echo $running_docker_status | sed -r 's/[\"]+/\\\"/g')
+		# cleaned_docker_status=$(python2 -c 'import sys, urllib; print urllib.quote(sys.argv[1])' "$running_docker_status")
+
+		# done
+		echo "{\"build\":\"busy\",\"process\":\"Output from Docker: ${cleaned_docker_status}\",\"timestamp\":\"$( date +%s )\"}"  > /var/goddard/build.json
```

Comment:
@jonathanendersby it's here for that ticket... i don't see any messages today though.